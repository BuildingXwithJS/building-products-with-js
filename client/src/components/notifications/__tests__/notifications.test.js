// npm packages
import configureMockStore from 'redux-mock-store';

// our packages
import Notifications from '../notifications';

// create mock store
const mockStore = configureMockStore();

test('# Notifications', () => {
  const store = mockStore({notifications: []});
  const wrapper = shallow(<Notifications store={store} />);
  expect(wrapper).toMatchSnapshot();
});
