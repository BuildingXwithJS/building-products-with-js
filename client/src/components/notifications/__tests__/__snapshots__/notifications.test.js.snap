exports[`test # Notifications 1`] = `
<Notifications
  dispatch={[Function]}
  notifications={Array []}
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  } />
`;
